src/patternfly/components/Login/login.scss

```diff
@@ -0,0 +1,245 @@
+    padding-top: var(--pf-c-login__main-header--PaddingTop);
+  }
+
+  // If the last child isn't the footer, then we need to put the footer's bottom padding there
+  > :last-child:not(.pf-c-login__main-footer) {
+    padding-bottom: var(--pf-c-login__main-footer--PaddingBottom);
+  }
+
+  // This is not mobile first but avoids removing the margin for larger devices
+  @media (max-width: $pf-global--breakpoint--xl) {
+    margin-bottom: var(--pf-c-login__main--MarginBottom);
+  }
+}
+
+.pf-c-login__main-header {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  column-gap: var(--pf-c-login__main-header--ColumnGap);
+  row-gap: var(--pf-c-login__main-header--RowGap);
+  flex: 0 0 auto;
+  align-items: center;
+  padding-top: var(--pf-c-login__main-header--PaddingTop);
+  padding-right: var(--pf-c-login__main-header--PaddingRight);
+  padding-bottom: var(--pf-c-login__main-header--PaddingBottom);
+  padding-left: var(--pf-c-login__main-header--PaddingLeft);
+
+  .pf-c-title {
+    grid-column: -1 / 1;
+  }
+}
+
+.pf-c-login__main-body {
+  flex: 1 1 auto;
+  padding-top: var(--pf-c-login__main-body--PaddingTop);
+  padding-right: var(--pf-c-login__main-body--PaddingRight);
+  padding-bottom: var(--pf-c-login__main-body--PaddingBottom);
+  padding-left: var(--pf-c-login__main-body--PaddingLeft);
+
+  .pf-c-form__group.pf-m-action {
+    display: grid;
+    justify-items: center;
+    grid-gap: var(--pf-c-login__main-body-form-group--m-action--GridGap);
+
+    @media (min-width: $pf-global--breakpoint--md) {
+      grid-template-columns: auto 1fr;
+      justify-items: start;
+    }
+
+    .pf-c-button {
+      --pf-c-button--PaddingLeft: var(--pf-c-login__main-body-form-group--m-action__button--PaddingX);
+      --pf-c-button--PaddingRight: var(--pf-c-login__main-body-form-group--m-action__button--PaddingX);
+
+      justify-self: stretch;
+    }
+
+    .pf-c-check {
+      display: flex;
+      align-items: center;
+    }
+  }
+}
+
+.pf-c-login__main-footer {
+  flex: 0 0 auto;
+  padding-top: var(--pf-c-login__main-footer--PaddingTop);
+  padding-right: var(--pf-c-login__main-footer--PaddingRight);
+  padding-bottom: var(--pf-c-login__main-footer--PaddingBottom);
+  padding-left: var(--pf-c-login__main-footer--PaddingLeft);
+
+  .pf-c-title {
+    margin-bottom: var(--pf-c-login__main-footer-title--MarginBottom);
+    text-align: center;
+
+    @media (min-width: $pf-global--breakpoint--md) {
+      text-align: left;
+    }
+  }
+}
+
+.pf-c-login__main-footer-link {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-gap: var(--pf-c-login__main-footer-link--GridGap);
+  align-items: center;
+  padding-top: var(--pf-c-login__main-footer-link--PaddingX);
+  padding-bottom: var(--pf-c-login__main-footer-link--PaddingX);
+  color: var(--pf-c-login__main-footer-link--Color);
+
+  &:hover,
+  &.pf-m-hover {
+    color: var(--pf-c-login__main-footer-link--hover--Color);
+  }
+}
+
+.pf-c-login__main-footer-link-text {
+  display: none;
```

Comment:
It does look weird, mostly because the spacing this demo needs isn't achievable using the current List Grid implementation. I created [this issue](https://github.com/patternfly/patternfly-next/issues/803) to track it a while back. Maybe it's possible to achieve what we need without List Grid?

Reply:
Should we handle this in this PR, or in #803?

Reply:
@andybraren @matthewcarleton what do you think? ^^

Reply:
I'd say handle it in that issue later.

Reply:
👍